airflow/contrib/hooks/aws_glue_job_hook.py

```diff
@@ -0,0 +1,130 @@
+# -*- coding: utf-8 -*-
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+
+from airflow.exceptions import AirflowException
+from airflow.contrib.hooks.aws_hook import AwsHook
+import time
+
+
+class AwsGlueJobHook(AwsHook):
+    """
+    Interact with AWS Glue - create job, trigger, crawler
+
+    :param job_name: unique job name per AWS account
+    :type str
+    :param desc: job description
+    :type str
+    :param region_name: aws region name (example: us-east-1)
+    :type region_name: str
+    """
+
+    def __init__(self,
+                 job_name=None,
+                 desc=None,
+                 aws_conn_id='aws_default',
+                 region_name=None, *args, **kwargs):
+        self.job_name = job_name
+        self.desc = desc
+        self.aws_conn_id = aws_conn_id
+        self.region_name = region_name
+        super(AwsGlueJobHook, self).__init__(*args, **kwargs)
+
+    def get_conn(self):
+        conn = self.get_client_type('glue', self.region_name)
```

Comment:
I would change this in 
```suggestion
        return self.get_client_type('glue', self.region_name)
```
Using the `conn` variable does not really add value from my perspective.